src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -2072,7 +2072,9 @@ private bool OpenJsonFileFromPath(string fileContents, string filePath, bool for
             catch (Exception e)
             {
                 Console.WriteLine(e.Message);
+#pragma warning disable CA2200 // Rethrow to preserve stack details
                 throw e;
+#pragma warning restore CA2200 // Rethrow to preserve stack details
```

Comment:
@sm6srw the documentation for `CA2200` says `To keep the original stack trace information with the exception, use the throw statement without specifying the exception.` I therefore believe that we can avoid this warning if we just say `throw` here, what do you think, or is the same exception needed to be thrown?